packages/components/organisms/f-cookie-banner/package.json

```diff
@@ -47,7 +47,10 @@
   },
   "peerDependencies": {
     "@justeat/browserslist-config-fozzie": ">=1.1.1",
-    "js-cookie": "2.2.1"
+    "js-cookie": "2.2.1",
+    "@justeat/f-button": "3.x.x",
```

Comment:
I think the syntax is `3.x` though I haven't tested to see what happens with the extra `x`.

Reply:
Should work but makes sense to do the `3.x` as that's the same as `3.x.x` syntax i believe. Updated it now.